index.html

```diff
@@ -6,9 +6,9 @@
 
     <link rel="shortcut icon" href="favicon.ico">
 
-    <link type="text/css" rel="stylesheet" href="../dhis-web-commons/material-design-icons/material-icons.css">
-    <link type="text/css" rel="stylesheet" href="../dhis-web-commons/fonts/roboto.css">
-    <link type="text/css" rel="stylesheet" href="../dhis-web-commons/font-awesome/css/font-awesome.min.css">
+    <link type="text/css" rel="stylesheet" href="/dhis-web-commons/material-design-icons/material-icons.css">
+    <link type="text/css" rel="stylesheet" href="/dhis-web-commons/fonts/roboto.css">
+    <link type="text/css" rel="stylesheet" href="/dhis-web-commons/font-awesome/css/font-awesome.min.css">
 
```

Comment:
Wouldn't this fail when the instance is namespaced? i.e. https://play.dhis2.org/demo